config/locales/en.yml

```diff
@@ -104,6 +104,7 @@ en:
         offence: 'Offence'
         court: 'Court'
         case_number: 'Case number'
+        providers_ref: 'Provider reference number'
```

Comment:
it is  - the tests complain about there being a missing key.  However, I've put them into common.external_user as suggested